auditlog/models/http_request.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# © 2015 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+from openerp.http import request
+
+
+class AuditlogHTTPRequest(models.Model):
+    _name = 'auditlog.http.request'
+    _description = u"Auditlog - HTTP request log"
+    _order = "create_date DESC"
+    _rec_name = 'display_name'
+
+    display_name = fields.Char(u"Name", compute="_display_name")
```

Comment:
Is this a blocking point to migrate on 9.0? At best rename `_display_name` to `_compute_display_name`.